config/zones/CA-ON.yaml

```diff
@@ -211,8 +226,7 @@ parsers:
   price: CA_ON.fetch_price
   production: CA_ON.fetch_production
 sources:
-  ? "Mallia, E., Lewis, G. \"Life cycle greenhouse gas emissions of electricity generation\
-    \ in the province of Ontario, Canada.\" Int J Life Cycle Assess 18, 377\u2013\
-    391 (2013)"
+  ? Mallia, Eric, and Geoffrey Lewis. "Life cycle greenhouse gas emissions of electricity generation in the province of Ontario, Canada."
```

Comment:
This is not formatted correctly, it should follow MLA citation style if I'm not wrong.

Also if you wrap the full key in `''` there is no need to escape double quotes or colons. It should also be on one line as right now it's interpreted as a explicit key with no value and an inplicit key with a explicit value.